vm/src/builtins/genericalias.rs

```diff
@@ -39,17 +39,15 @@ impl PyValue for PyGenericAlias {
     }
 }
 
-#[derive(FromArgs)]
-pub struct GenericAliasArgs {
-    origin: PyTypeRef,
-    arguments: PyObjectRef,
-}
-
 impl Constructor for PyGenericAlias {
-    type Args = GenericAliasArgs;
+    type Args = FuncArgs;
 
     fn py_new(cls: PyTypeRef, args: Self::Args, vm: &VirtualMachine) -> PyResult {
-        PyGenericAlias::new(args.origin, args.arguments, vm).into_pyresult_with_type(vm, cls)
+        if !args.kwargs.is_empty() {
+            return Err(vm.new_type_error("GenericAlias() takes no keywords".to_owned()));
```

Comment:
```suggestion
            return Err(vm.new_type_error("GenericAlias() no keyword arguments".to_owned()));
```
Although it is trivial, this might be more precise.

Reply:
Ugh, don't know how I missed that. Thanks!